model/schema/sssom.yaml

```diff
@@ -22,6 +22,11 @@ default_curi_maps:
 default_prefix: sssom
 
 enums:
+  predicate_modifier_enum:
+    permissible_values:
+      Not: Negating the mapping predicate. The meaning of the triple becomes subject_id is not a predicate_id match to object_id.
+      Direct: States that the mapping predicate directly relates two entities connected by a transitive predicate like rdfs:subClassOf, i.e. there is no other entities in between.
+      Inverse: Inverting the predicate. The meaning of the triple becomes subject_id is inverse-related with predicate_id match to object_id (effectively turning the direction around).
```

Comment:
I think we explicitly agreed not to add direct/inverse as predicate modifiers

Reply:
or at least to table that discussion for later

Reply:
So we only add "Not" at the moment? I thought @cmungall main use case were Inverse and Direct.

Reply:
I removed Indirect and Inverse for now - we can add them later. One evil at a time.